### ISSUE TITLE
Cannot get netdata-speedtest to work

### ISSUE THREAD

**Issue by oksnfjan** [ohthehugemanatee/netdata-speedtest#8]
Running: netdata v1.43.0-124-nightly
OS: Ubuntu Focal (running in WSL2 on Windows 10)

I have followed the steps in https://blog.netdata.cloud/speedtest-monitoring/, but it does not seem to work. Looking at my network load in Task Manager, I can see no load great enough to be a speed test when Netdata is running.

However, it runs when I execute “/usr/libexec/netdata/plugins.d/charts.d.plugin” as the netdata user. I can also see that the speedtest.conf file is loaded, as a change in the update frequency is reflected in the bash when running charts.d.plugin. I have tried to set the update frequency as low as 40 seconds, and still, no data shows up in the Netdata “Overview”.

Any tips?

**Comment by oksnfjan** [comment]
Update:

TLDR: Should --accept-license be added to the speedtest_get() function?
`output=$(speedtest --accept-license --format=csv)`


Looking at /var/log/netdata/collector.log, it was clear that charts.d failed due to an error in speedtest.chart.sh. Even if I had executed 

`sudo -u netdata speedtest`

and accepted the license (and it works fine when I run "/usr/libexec/netdata/plugins.d/charts.d.plugin"), it still complained:

`To accept the message please run speedtest interactively or use the following:
    speedtest --accept-license
/usr/libexec/netdata/plugins.d/../charts.d/speedtest.chart.sh: line 46: *8/1000: syntax error: operand expected (error token is "*8/1000")`

So I added --accept-license in the speedtest_get() function of speedtest.chart.sh
`output=$(speedtest --accept-license --format=csv)`

And now I can see that the collector is running as expected. And now I see the chart in Netdata Overview!
